docs/mimimal-softlayer.yml

```diff
@@ -366,6 +307,62 @@ jobs:
   properties:
     login:
       catalina_opts: -Xmx768m -XX:MaxPermSize=256m
+      saml:
+        serviceProviderCertificate: |
+          -----BEGIN CERTIFICATE-----
+          MIIENjCCAh6gAwIBAgIQKKBo+285koaiAaJKVNstEzANBgkqhkiG9w0BAQsFADAZ
```

Comment:
Note that https://github.com/cloudfoundry/cf-release/blob/master/example_manifests/minimal-aws.yml contains sample manifests too. We tried to stick with that one as close as possible.